apps/website/screens/principles/data-visualization/DataVisualizationPage.tsx

```diff
@@ -0,0 +1,345 @@
+        <DxcBulletedList>
+          <DxcBulletedList.Item>
+            <Link href="https://cloudscape.design/components/bar-chart/" passHref legacyBehavior>
+              <DxcLink>Bar chart</DxcLink>
+            </Link>
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <Link href="https://cloudscape.design/components/line-chart/" passHref legacyBehavior>
+              <DxcLink>Line chart</DxcLink>
+            </Link>
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <Link href="https://cloudscape.design/components/mixed-line-bar-chart/" passHref legacyBehavior>
+              <DxcLink>Mixed line and bar chart</DxcLink>
+            </Link>
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <Link href="https://cloudscape.design/components/area-chart/" passHref legacyBehavior>
+              <DxcLink>Area chart</DxcLink>
+            </Link>
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            <Link href="https://cloudscape.design/components/pie-chart/" passHref legacyBehavior>
+              <DxcLink>Pie and donut chart</DxcLink>
+            </Link>
+          </DxcBulletedList.Item>
+        </DxcBulletedList>
+        <DxcParagraph>
+          All the information presented below complements the{" "}
+          <Link href="https://cloudscape.design/patterns/general/data-vis/" passHref legacyBehavior>
+            <DxcLink>Cloudscape data visualization pattern</DxcLink>
+          </Link>
+          . We strongly recommend reading it in parallel.
+        </DxcParagraph>
+      </>
+    ),
+  },
+  {
+    title: "Bar chart",
+    content: (
+      <DxcParagraph>
+        A bar chart is a graphical representation that displays and compares discrete data categories using rectangular
+        bars. Each bar's length or height is proportional to the frequency or value of its corresponding category,
+        allowing users to identify which groups are the highest or most common and compare them with others.
+      </DxcParagraph>
+    ),
+    subSections: [
+      {
+        title: "Do's",
+        content: (
+          <DxcBulletedList>
+            <DxcBulletedList.Item>
+              Ensure both the X and Y axes are labeled with appropriate units and descriptions.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Always plot bars against a zero-value baseline. It simplifies bar length comparison and ensures accurate
+              data visualization. A non-zero baseline can distort the comparison as it disrupts the ratio between bar
+              lengths and actual values.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              When constructing a bar chart, consider the order in which you will plot the bars. A common convention is
+              to sort the bars from longest to shortest. While comparisons can be made regardless of order, this
```

Comment:
"regardless of the order" feels more accurate than "regardless of order" (Even though as far as I know, both are correct)